models/cm_default.nestml

```diff
@@ -0,0 +1,196 @@
+"""
+Example compartmental model for NESTML
+
+Description
++++++++++++
+Corresponds to standard compartmental model implemented in NEST.
+
+References
+++++++++++
+
+
+See also
+++++++++
+
+
+Author
+++++++
+Willem Wybo
+"""
+neuron cm_default:
+
+  state:
+
+    # the presence of the state variable [v_comp]
+    # triggers compartment model context
+    v_comp real = 0
+
+    ### ion channels ###
+    # initial values state variables sodium channel
+    m_Na real = 0.0
+    h_Na real = 0.0
+
+    # initial values state variables potassium channel
+    n_K real = 0.0
+
+  end
+
+
+  parameters:
+    ### ion channels ###
+    # default parameters sodium channel
+    e_Na real = 50.0
+    gbar_Na real = 0.0
+
+    # default parameters potassium channel
+    e_K real = -85.0
+    gbar_K real = 0.0
+
+    ### synapses ###
+    e_AMPA real = 0 mV            # Excitatory reversal Potential
+    tau_r_AMPA real = 0.2 ms      # Synaptic Time Constant Excitatory Synapse
+    tau_d_AMPA real = 3.0 ms      # Synaptic Time Constant Excitatory Synapse
+
+    e_GABA real = -80. mV            # Inhibitory reversal Potential
+    tau_r_GABA real = 0.2 ms      # Synaptic Time Constant Inhibitory Synapse
+    tau_d_GABA real = 10.0 ms      # Synaptic Time Constant Inhibitory Synapse
+
+    e_NMDA real = 0 mV            # NMDA reversal Potential
+    tau_r_NMDA real = 0.2 ms      # Synaptic Time Constant NMDA Synapse
+    tau_d_NMDA real = 43.0 ms      # Synaptic Time Constant NMDA Synapse
+
+    e_AN_AMPA real = 0 mV            # Excitatory reversal Potential
+    tau_r_AN_AMPA real = 0.2 ms      # Synaptic Time Constant Excitatory Synapse
+    tau_d_AN_AMPA real = 3.0 ms      # Synaptic Time Constant Excitatory Synapse
+    e_AN_NMDA real = 0 mV            # NMDA reversal Potential
+    tau_r_AN_NMDA real = 0.2 ms      # Synaptic Time Constant NMDA Synapse
+    tau_d_AN_NMDA real = 43.0 ms     # Synaptic Time Constant NMDA Synapse
+    NMDA_ratio real = 2.0      # NMDA_ratio
+  end
+
+  equations:
+    """
```

Comment:
Please change this into a multiline comment prepended by ``#`` on each line, rather than ``"""``.